src/google/protobuf/descriptor.ts

```diff
@@ -1810,100 +2002,116 @@ export const FieldDescriptorProto = {
       writer.uint32(10).string(message.name);
     }
+
     if (message.number !== 0) {
       writer.uint32(24).int32(message.number);
     }
-    if (message.label !== 1) {
+
+    if (message.label !== 0) {
       writer.uint32(32).int32(message.label);
     }
-    if (message.type !== 1) {
+
+    if (message.type !== 0) {
       writer.uint32(40).int32(message.type);
     }
+
     if (message.typeName !== "") {
       writer.uint32(50).string(message.typeName);
     }
+
     if (message.extendee !== "") {
       writer.uint32(18).string(message.extendee);
     }
+
     if (message.defaultValue !== "") {
       writer.uint32(58).string(message.defaultValue);
     }
+
     if (message.oneofIndex !== 0) {
       writer.uint32(72).int32(message.oneofIndex);
     }
+
     if (message.jsonName !== "") {
       writer.uint32(82).string(message.jsonName);
     }
+
     if (message.options !== undefined) {
       FieldOptions.encode(message.options, writer.uint32(66).fork()).ldelim();
     }
-    if (message.proto3Optional === true) {
-      writer.uint32(136).bool(message.proto3Optional);
-    }
+
     return writer;
   },
 
   decode(input: _m0.Reader | Uint8Array, length?: number): FieldDescriptorProto {
     const reader = input instanceof _m0.Reader ? input : new _m0.Reader(input);
     let end = length === undefined ? reader.len : reader.pos + length;
     const message = createBaseFieldDescriptorProto();
+
     while (reader.pos < end) {
       const tag = reader.uint32();
+
       switch (tag >>> 3) {
         case 1:
           message.name = reader.string();
           break;
+
         case 3:
           message.number = reader.int32();
           break;
+
         case 4:
           message.label = reader.int32() as any;
           break;
+
         case 5:
           message.type = reader.int32() as any;
           break;
+
         case 6:
           message.typeName = reader.string();
           break;
+
         case 2:
           message.extendee = reader.string();
           break;
+
         case 7:
           message.defaultValue = reader.string();
           break;
+
         case 9:
           message.oneofIndex = reader.int32();
           break;
+
         case 10:
           message.jsonName = reader.string();
           break;
+
         case 8:
           message.options = FieldOptions.decode(reader, reader.uint32());
           break;
-        case 17:
-          message.proto3Optional = reader.bool();
-          break;
+
         default:
           reader.skipType(tag & 7);
           break;
       }
     }
+
     return message;
   },
 
   fromJSON(object: any): FieldDescriptorProto {
     return {
       name: isSet(object.name) ? String(object.name) : "",
       number: isSet(object.number) ? Number(object.number) : 0,
-      label: isSet(object.label) ? fieldDescriptorProto_LabelFromJSON(object.label) : 1,
-      type: isSet(object.type) ? fieldDescriptorProto_TypeFromJSON(object.type) : 1,
```

Comment:
this is an old proto2 (enums that start from 1). Not sure we need to support this and this file in generally is not used.